libs/feature/search/src/lib/utils/links/link-helper.service.ts

```diff
@@ -78,4 +79,13 @@ export class LinkHelperService {
   hasProtocolDownload(link: MetadataLinkValid): boolean {
     return /^WWW:DOWNLOAD/.test(link.protocol)
   }
+
+  getLinkLabel(link: MetadataLinkValid): string {
+    let source = getFileFormat(link)
+    if (this.isWmsLink(link)) source = 'WMS'
```

Comment:
`else`
It's not clear what have the priority to what.

Reply:
ok, I'll adapt